### PR TITLE
refactor(util.sh): removal of duplicate code in hack/cstor_volume/test/util.sh

### LINKED ISSUE #730: refactor code in /hack/cstor_volume/test/util.sh
## Duplicate code in /hack/cstor_volume/test/util.sh

* Is this a BUG REPORT or FEATURE REQUEST?
 FEATURE REQUEST

**What happened**:
Duplicate code in kubectlApply() and kubectlDelete() functions.

**What you expected to happen**:
Code reusability at its best (with least complexity, of course)

**How to reproduce it (as minimally and precisely as possible)**:
No need to reproduce it

**Anything else we need to know?**:
I'm working on it.  Refactoring to create a function with common code in util.sh file

### PR DESCRIPTION
**Pull request by ferozbaig96**
**What this PR does / why we need it**:
Duplicate code in kubectlApply() and kubectlDelete() functions.
Refactored code to create a function 'prefixedFileNames()' with common code in util.sh file to ensure code reusability at its best (with least complexity, of course)

**Which issue this PR fixes**: fixes #730 

**Special notes for your reviewer**: Please label it with hacktoberfest label.

### CODE REVIEW

**Comment by ferozbaig96** [comment]
@princerachit please have a look at it and let me know in case of any issues.

File: hack/cstor_volume/test/util.sh [hunk 1]
```diff
@@ -1,28 +1,28 @@
...
     # append all the files together prefixed with -f
-    filesToApply=""
+    local filesToApply=""
     while [ "$1" != "" ]; do
         filesToApply="$filesToApply -f $1"
         sleep 1
         shift
     done
 
+    echo $filesToApply
```
Comment (princerachit) [inline hack/cstor_volume/test/util.sh]: Can you add some informative comment in echo. printing `-f abc -f ghh -f ggg` does not make any sense.

Comment (ferozbaig96) [reply hack/cstor_volume/test/util.sh]: Done with the requested changes. Please have a look.

**Comment by princerachit** [comment]
The dco sign is failing. Please fix it.

**Comment by ferozbaig96** [comment]
> The dco sign is failing. Please fix it.

Done

**Pull request merged**